setup.cfg

```diff
@@ -25,7 +25,10 @@ classifiers =
 requires = numpy
 zip_safe = False
 tests_require = pytest-astropy
-setup_requires = numpy>=1.16
+setup_requires =
```

Comment:
Isn't it easier to pin a minversion of `pip`? 😬

Reply:
No because by the time this gets parsed pop is already actively in use so any pinning would have no effect